### PR TITLE
US-3.13: Refactoring event drawer CSS

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/component/Drawer/Style.less [hunk 1]
```diff
@@ -96,4 +96,566 @@
       min-width: auto;
     }
   }
-}
\ No newline at end of file
+}
+
+/*=============== Event drawer component ===========*/
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/component/Drawer/Style.less]: All these styles are specific to calendar, so they should be moved to a dedicated file in the calendar folder IMO, this file should only contain the style of the generic drawer component.

Comment (AzmiTouil) [reply platform-ui-skin/src/main/webapp/skin/less/component/Drawer/Style.less]: Done

File: platform-ui-skin/src/main/webapp/skin/less/Core.less [hunk 2]
```diff
@@ -43,4 +43,5 @@
 @import "Oauth/Style.less";
 @import "_global.less";
 @import "_responsive.less";
+@import "calendar/EventDrawer/Style.less";
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/Core.less]: It should not be imported in the Core style since it is specific to calendar.

Comment (AzmiTouil) [reply platform-ui-skin/src/main/webapp/skin/less/Core.less]: Yes, I know it's specific to the calendar, but I have to reuse it in the digital workplace,so how can this be done ?

Comment (thomasdelhomenie) [reply platform-ui-skin/src/main/webapp/skin/less/Core.less]: You can load the calendar stylesheet when your portlet is displayed, in gatein-resources.xml:
```
  <portlet-skin>
    <application-name>myApplication</application-name>
    <portlet-name>myPortlet</portlet-name>
    <skin-name>Enterprise</skin-name>
    <css-path>/skin/css/calendar/calendar.css</css-path>
    <css-priority>1</css-priority>
  </portlet-skin>
```

Comment (AzmiTouil) [reply platform-ui-skin/src/main/webapp/skin/less/Core.less]: Done,thanks @thomasdelhomenie

**Pull request closed without merging**